monitoring/tsconfig.eslint.json

```diff
@@ -1,5 +1,5 @@
 {
 	"extends": "./tsconfig.json",
-	"include": ["src/**/*", "*.ts", "node_modules/puppeteer-core/**/*"],
+	"include": ["src/**/*", "*.ts", "node_modules/playwright-core/**/*"],
```

Comment:
Why are we type-checking playwright?

Reply:
Just swapped it over but have removed it.